### DISCUSSION TITLE
Scheduler crashing in Dynamic task mapping

### DISCUSSION THREAD

**Discussion by saurabhkhera** [apache/airflow#36302] [Q&A]
I would like to inquire about the status of the reported issue in the following ticket: https://github.com/apache/airflow/issues/31292#issuecomment-1634509601. If the issue has been resolved, could you please specify the version in which it was addressed? Despite my efforts, I haven't been able to locate this information. If anyone has insights on this matter, your assistance would be greatly appreciated.

**Comment by saurabhkhera** [comment]
@eladkal @potiuk any insights on this issue?

**Reply by potiuk** [reply]
There is a linked issue/ PR in the comments that has version added. just follow that.

Not sure why you are calling me or Elad for that - we have to follow the same issues and comments as you - this is all public. and we cannot authoritatively answer if your issue is solved there, it might be similar or the same or completely different. 

But my best advise is always the same - upgrade to latest version and see if your problem is fixed if you suspect to do so if you cannot be 100% sure from the description of issue and changelog. This is always the best way. There is no other way to get the fix than to upgrade.

If you upgrade and it gets fixed, you are done (and this was the best way). 

If you upgrade and it is not fixed - you can gather evidences and open a new issue explaining your problem, hoping that the evidences you provide will be enough for someone to take a look and maybe even fix it. Or not - there are no guarantees for that. 

This is open-source and software that you pay exactly 0 for, you as a user. need to take the burden and risk of spending time and effort on upgrading, hoping that the issue will be fixed, but it's rarely you can be 100% sure, nor anyone can give you authoritative answer. You (or your compnay) can also pay consultants to get more of the certainty to analyse your precise case. But if you don't then part of choosing open-source software instead of managed software or propriatary software that you pay for support of  is that some of the burden on maintenance and upgrades is on your shoulders (at the expense of not paying for it).
